src/Runtime/Distributions/DiscreteChar.cs

```diff
@@ -505,7 +506,7 @@ public bool IsUniform()
         {
             foreach (var range in this.Data.Ranges)
             {
-                if (Math.Abs(range.Probability - UniformProb) > Eps)
+                if (Math.Abs(range.Probability.LogValue - UniformProb.LogValue) > Eps)
```

Comment:
I think it is probably OK, but as we know the scale change from UniformProb to UniformProb.LogValue, should we be changing Eps here?

Reply:
Actually, this is quite safe because below one (i.e. with any discrete probabilities) the scale change is always > 1 and so any deltas will be accentuated.